service/policy/db/migrations/20250909000000_add_registered_peps_to_obligations_triggers.sql

```diff
@@ -0,0 +1,16 @@
+-- +goose Up
+-- +goose StatementBegin
+-- Add client_id column to obligation_triggers table
+ALTER TABLE IF EXISTS obligation_triggers
+ADD COLUMN IF NOT EXISTS client_id TEXT NOT NULL;
```

Comment:
Can we please allow this to be null in both the proto and schema so scoping a trigger to a PEP client ID is optional and not required?